apps/settings/js/panels/languages/languages.js

```diff
@@ -41,15 +41,6 @@ define(function(require) {
           hour: 'numeric',
           minute: 'numeric'
         });
-
-        if (navigator.language in navigator.mozL10n.qps) {
-          timeString =
-            navigator.mozL10n.qps[navigator.language].translate(timeString);
-          dateString =
-            navigator.mozL10n.qps[navigator.language].translate(dateString);
-        }
```

Comment:
@zbraniecki Can you review this change?  I'd like the time and date samples in Settings to show plain French strings coming from `Intl`, so that it's consistent with whot it's going to look like in other places.